common/test/acceptance/tests/test_cohorted_courseware.py

```diff
@@ -177,31 +260,87 @@ def login_and_verify_visible_problems(username, email, expected_problems):
+            [self.group_verified_problem, self.visible_to_all_problem],
+            'verified'
+        )
+
+        login_and_verify_visible_problems(
+            self.cohort_audit_student_username, self.cohort_audit_student_email,
+            [self.group_audit_problem, self.visible_to_all_problem],
+            'audit'
+        )
+
+        login_and_verify_visible_problems(
             self.cohort_default_student_username, self.cohort_default_student_email,
-            [self.visible_to_all_problem]
+            [self.visible_to_all_problem],
         )
 
     def test_cohorted_courseware(self):
         """
         Scenario: Can create content that is only visible to students in particular cohorts
-          Given that I have course with 4 problems, 1 staff member, and 3 students
+          Given that I have course with 6 problems, 1 staff member, and 6 students
           When I enable cohorts in the course
+          And I add the Course Modes for Verified and Audit
           And I create two content groups, Content Group A, and Content Group B, in the course
           And I link one problem to Content Group A
           And I link one problem to Content Group B
+          And I link one problem to Content Group Verified
+          And I link one problem to Content Group Audit
           And I link one problem to both Content Group A and Content Group B
           And one problem remains unlinked to any Content Group
           And I create two manual cohorts, Cohort A and Cohort B,
             linked to Content Group A and Content Group B, respectively
           And I assign one student to each manual cohort
+          And I assign one student to each enrollment track
           And one student remains in the default cohort
-          Then the staff member can see all 4 problems
-          And the student in Cohort A can see all the problems except the one linked to Content Group B
-          And the student in Cohort B can see all the problems except the one linked to Content Group A
+          Then the staff member can see all 6 problems
+          And the student in Cohort A can see all the problems linked to A
+          And the student in Cohort B can see all the problems linked to B
+          And the student in Verified can see the problems linked to Verified and those not linked to a Group
+          And the student in Audit can see the problems linked to Audit and those not linked to a Group
           And the student in the default cohort can ony see the problem that is unlinked to any Content Group
         """
         self.enable_cohorting(self.course_fixture)
         self.create_content_groups()
         self.link_problems_to_content_groups_and_publish()
         self.create_cohorts_and_assign_students()
         self.view_cohorted_content_as_different_users()
+
+
+class EnrollmentTrackModeTest(ContainerBase):
+
+    def setUp(self, is_staff=True):
```

Comment:
Can you add just a brief comment to this test (don't need to do the BDD style) stating that the purpose of these tests are to ensure that Enrollment Tracks only show on the Group Configurations page if there are > 1 tracks?